core/src/main/java/io/undertow/client/http/HttpClientConnection.java

```diff
@@ -79,11 +79,6 @@
 import java.util.concurrent.CopyOnWriteArrayList;
 
 import static io.undertow.client.UndertowClientMessages.MESSAGES;
-import static io.undertow.util.Headers.CLOSE;
-import static io.undertow.util.Headers.CONNECTION;
-import static io.undertow.util.Headers.CONTENT_LENGTH;
```

Comment:
Why did you remove all the static imports?

Reply:
These static imports are no longer required. For example, CLOSE is accessible as Headers.CLOSE.

Reply:
Ah, I just got confused because there were not replaced with the equivalent static imports, my bad.